pom.xml

```diff
@@ -115,6 +115,8 @@
         <!-- Camel target version -->
         <camel-version>4.8.0.redhat-00005</camel-version>
 
+        <camel-kamelets-version>4.8.0</camel-kamelets-version>
```

Comment:
will this one get aligned by PNC with productized version?

Reply:
Yes - build ordering is always camel-kamelets before camel-spring-boot https://gitlab.cee.redhat.com/fuse-prod/build-configurations/-/blob/master/camel/4.8.0/build-config.yaml?ref_type=heads#L62-73